packages/libs/react-components/src/components/Card/styles.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { styled } from '../../styles';
+
+export const colorVariant = {
```

Comment:
I would suggest to use theme: dark | light instead of default and negative as variants to keep it consistent